docs/docs/architecture/attestation.md

```diff
@@ -220,6 +220,46 @@ The latter means that the value can be generated offline and compared to the one
 </tabItem>
 </tabs>
 
+### CVM verification
+
+To verify runtime measurements, a chain of trust from the CVM technology to the interface providing the measurements has to be established.
+For verification of the CVM technology, Constellation may expose additional options in its config file.
+
+<tabs groupId="csp">
+<tabItem value="azure" label="Azure SEV-SNP">
+
+On Azure, AMD SEV-SNP is used to provide runtime encryption to the VMs.
+An SEV-SNP attestation report is also used to establish trust in the vTPM running in the VM.
+Some parameters for verification of the SEV-SNP attestation statement can be customized by the user using the Constellation config file.
+
+* TCB versions:
+
+  The minimum version numbers of components in the SEV-SNP TCB may set to enforce only machines with the most recent microcode version are allowed to join the cluster.
+  Alternatively, this may also be used to set a lower minimum version to allow slightly out-of-date machines to still be able to join the cluster.
+
+* AMD Root Key Certificate:
+
+  The certificate used as a root of trust to verify the SEV-SNP certificate chain.
+
+* Firmware Signer
```

Comment:
```suggestion
* TCB versions

  The minimum version numbers of components in the SEV-SNP TCB may set to enforce only machines with the most recent microcode version are allowed to join the cluster.
  Alternatively, this may also be used to set a lower minimum version to allow slightly out-of-date machines to still be able to join the cluster.

* AMD Root Key Certificate

  The certificate used as a root of trust to verify the SEV-SNP certificate chain.

* Firmware Signer
```
nit: either end all items with colons or none. I'd prefer none.